front/src/pages/Overview/index.js

```diff
@@ -13,7 +13,7 @@ class Overview extends React.Component {
         <h2>{i18n.t('useful_links')}</h2>
         <p>
           {i18n.t('suggestion')}{' '}
-          <a href="mailto:contact-upsignon-rgsystem@septeo.com">
+          <a href="mailto:BS-SEPTEOITSOLUTIONS-Support@septeo.comepBS-SEPTEOITSOLUTIONS-Support@septeo.com
```

Comment:
I think this is not good
```suggestion
          <a href="mailto:BS-SEPTEOITSOLUTIONS-Support@septeo.com">
```